dask_cuda/__init__.py

```diff
@@ -3,7 +3,6 @@
 if sys.platform != "linux":
     raise ImportError("Only Linux is supported by Dask-CUDA at this time")
 
-
```

Comment:
I'm surprised linting didn't complain about this. It doesn't matter so let's keep it like this.